contrib/easyconfig-templates/2023a/PythonBundle-scipy.eb.tmpl

```diff
@@ -0,0 +1,39 @@
+# Template for a python bundle using SciPy-bundle on 2023a
```

Comment:
can we add the toolchain name and version to the file name?

Reply:
I thought about this, but it is very messy because the names of the templates still need to have some description of what type of installation they cover. And mixing both results in very long filenames that are difficult to read.

Moreover, since the template has to be renamed in any case, there is little gain by already having the toolchain in the filename.